Assets/ZSSRichTextEditor.js

```diff
@@ -274,8 +272,12 @@ ZSSEditor.getSelectedText = function() {
 ZSSEditor.getCaretArguments = function() {
     var caretInfo = this.getYCaretInfo();
     
-    this.caretArguments[0] = 'yOffset=' + caretInfo.y;
-    this.caretArguments[1] = 'height=' + caretInfo.height;
+    if (caretInfo == null) {
+        this.caretArguments = null;
+    } else {
+        this.caretArguments[0] = 'yOffset=' + caretInfo.y;
```

Comment:
If `caretArguments` was ever set to null then this would break. Maybe we need to add `this.caretArguments = [];` above this line?